template/{{app_name}}/app/helpers/uswds_form_builder.rb

```diff
@@ -59,49 +72,75 @@ def radio_button(attribute, tag_value, options = {})
   end
 
   def select(attribute, choices, options = {}, html_options = {})
-    classes = "usa-select"
-
-    html_options[:class] ||= ""
-    html_options[:class].prepend("#{classes} ")
+    append_to_option(html_options, :class, " usa-select")
 
     label_text = options.delete(:label)
 
-    us_form_group(attribute: attribute) do
+    form_group(attribute) do
       us_text_field_label(attribute, label_text, options) + super(attribute, choices, options, html_options)
     end
   end
 
   def submit(value = nil, options = {})
-    options[:class] ||= ""
-    options[:class].prepend("usa-button ")
-
-    super(value, options)
-  end
-
-  def honeypot_field
-    spam_trap_classes = "opacity-0 position-absolute z-bottom top-0 left-0 height-0 width-0"
-    label_text = "Do not fill in this field. It is an anti-spam measure."
+    append_to_option(options, :class, " usa-button")
 
-    @template.content_tag(:div, class: "usa-form-group #{spam_trap_classes}") do
-      label(:spam_trap, label_text, { tabindex: -1, class: "usa-label #{spam_trap_classes}" }) +
-      @template.text_field(@object_name, :spam_trap, { autocomplete: "false", tabindex: -1, class: "usa-input #{spam_trap_classes}" })
+    if options[:big]
+      append_to_option(options, :class, " usa-button--big margin-y-6")
     end
+
+    super(value, options)
   end
 
   ########################################
   # Custom helpers
   ########################################
 
+  def tax_id_field(attribute, options = {})
+    options[:inputmode] = "numeric"
+    options[:placeholder] = "_________"
+    options[:width] = "md"
+
+    # Actual USWDS mask functionality broken until this is fixed:
+    # https://github.com/uswds/uswds/issues/5517
+    # append_to_option(options, :class, " usa-masked")
+
+    append_to_option(options, :hint, @template.content_tag(:p, I18n.t("us_form_with.tax_id_format")))
+
+    text_field(attribute, options)
+  end
+
+  def date_picker(attribute, options = {})
+    raw_value = object.send(attribute) if object
+
+    append_to_option(options, :hint, @template.content_tag(:p, I18n.t("us_form_with.date_picker_format")))
```

Comment:
The content `us_form_with.date_picker_format` needs added, here https://github.com/navapbc/template-application-rails/blob/fc56ff84fd49a71e718f645b8d1b6bf15d8a9efc/template/%7B%7Bapp_name%7D%7D/config/locales/defaults/en.yml#L61

See https://github.com/navapbc/pfml-starter-kit-app/blob/a23df2a2c9cbb0b8fae8eb4c3799dd670c0baf74/paidleave/config/locales/defaults/en.yml#L75